reo/tests/test_cold.py

```diff
@@ -0,0 +1,120 @@
+import json
```

Comment:
@Bill-Becker this test has a lot of unused imports and some unused lines - and I seem to recall that this test was just for development but not for long-term use?  Should we keep it?

Reply:
We should keep it. It tests various aspects of cooling load and technologies, as well as the heating <-> cooling technology interaction. I've had a number of instances which broke this unexpectedly and it flagged the issue.